modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -0,0 +1,236 @@
+
+== Dependency Information
+
+----
+<dependency>
+  <groupId>org.mule.modules</groupId>
+  <artifactId>mule-asynckit-module</artifactId>
+  <version>LATEST_VERSION</version> //<!--1-->
+  <classifier>mule-plugin</classifier>
+</dependency>
+----
+[calloutlist]
+.. Check https://docs.mulesoft.com/release-notes/apikit/apikit-release-notes[APIkit release notes] to get the *LATEST_VERSION*.
+
+== Namespace and Schema
+
+----
+xmlns:apikit-asyncapi="http://www.mulesoft.org/schema/mule/apikit-asyncapi" 
+xsi:schemaLocation="http://www.mulesoft.org/schema/mule/apikit-asyncapi http://www.mulesoft.org/schema/mule/apikit-asyncapi/current/mule-apikit-asyncapi.xsd">
+----
+
+== Configurations
+
+=== Config
+
+Default configuration
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Name | String | Name for this configuration. Connectors reference the configuration with this name | None | *Yes*
+| Api Definition | String |  API definition | None | *Yes*
+| Kafka Configs | Array of <<KafkaConfig>> | None | No |  
+| Anypoint MQ Configs | Array of <<AnypointMqConfig>> | None | No |  
+| Name | String | Identifier of this element used to reference it in other components | None | *Yes*
+|===
+
+=== Supported Message Brokers
+
+[%header%autowidth.spread]
+|===
+| Feature | Description | Supported Protocols | Configuration Details 
+| Anypoint MQ Message Broker | Supports asynchronous messaging with Anypoint MQ. | anypointmq | Anypoint MQ configurations are specified in the server object of the AsyncAPI specification. Supports both publish and subscribe operations. 
+| Kafka Message Broker | Supports asynchronous messaging with Apache Kafka. | kafka, kafka-secure | Kafka configurations are specified in the server object of the AsyncAPI specification. Supports both publish and subscribe operations. 
+|===
+
+==== Associated Operations
+* <<Publish>>  
+
+==== Associated Sources
+* <<MessageListener>>  
+
+== Operations
+
+[[Publish]]
+== Publish
+`<apikit-asyncapi:publish>`
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use | None | *Yes* 
+| Content | Binary | None | #[payload] | No  
+| Config Ref | ConfigurationProvider | Name of the configuration used to execute this component | None | *Yes* 
+| Channel Name | String | Channel name | None | *Yes* 
+| Server Name | String | Server name | None | *Yes* 
+| Target Variable | String | Name of a variable on which the operation's output is placed | None | No 
+| Target Value | String | Expression that is evaluated against the operation's output and the effect of that expression is stored in the target variable | #[payload] | No
+| Error Mappings | Array of <<ErrorMapping>> | Set of error mappings | None | No
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+| *Type* | PublishResponse
+| *Attributes Type* | Any
+|===
```

Comment:
Attributes Type row is removed in the latest.

Reply:
OK! Thanks!